src/simulation.hpp

```diff
@@ -576,6 +579,8 @@ template <typename problem_t> void AMRSimulation<problem_t>::setInitialCondition
 		ReadCheckpointFile();
 	}
 
+	calculateGpotAllLevels(dt_[0]);
```

Comment:
@BenWibking what should be the argument passed to this function when it's called in the initial conditions?

Reply:
I don't think `dt` is actually used by the Poisson solver, so you can just get rid of it entirely.

Reply:
Ah yes I just realised that a minute ago!